src/material/autocomplete/autocomplete.spec.ts

```diff
@@ -3371,6 +3371,32 @@ describe('MatAutocomplete', () => {
 
     expect(fixture.componentInstance.trigger.panelOpen).toBe(true);
   });
+
+  it('should emit from `autocomplete.closed` after click outside inside the NgZone', fakeAsync(() => {
+    const inZoneSpy = jasmine.createSpy('in zone spy');
+
+    const fixture = createComponent(SimpleAutocomplete, [
+      {provide: NgZone, useFactory: () => new NgZone({enableLongStackTrace: false})},
+    ]);
+    const zone = TestBed.inject(NgZone);
+    fixture.detectChanges();
```

Comment:
This shouldn't be necessary. The zone is already stubbed out for all tests inside `createComponent`. Try searching for `simulateZoneExit` in the same file.

Reply:
> This shouldn't be necessary. The zone is already stubbed out for all tests inside `createComponent`. Try searching for `simulateZoneExit` in the same file.

 I have got real `NgZone` from DI, instead `MockZone` which get from function `CreateComponent`